mysql/tests/test_query_activity.py

```diff
@@ -159,6 +159,8 @@ def _run_blocking(conn):
 )
 def test_activity_reported_hostname(aggregator, dbm_instance, dd_run_check, reported_hostname, expected_hostname):
     dbm_instance['reported_hostname'] = reported_hostname
+    dbm_instance['read_timeout'] = 0.001
```

Comment:
run with 
ddev test mysql:py3.11-5.7 -- -k test_activity_reported_hostname -vvv tests/test_query_activity.py

pymysql.err.OperationalError: (2013, 'Lost connection to MySQL server during query (timed out)')

ERROR    datadog_checks.base.checks.base.mysql:mysql.py:327 error!
Traceback (most recent call last):
  File "/root/dd/integrations-core/mysql/datadog_checks/mysql/mysql.py", line 295, in check
    self.version = get_version(db)
                   ^^^^^^^^^^^^^^^
  File "/root/dd/integrations-core/mysql/datadog_checks/mysql/version_utils.py", line 16, in get_version
    cursor.execute('SELECT VERSION()')
  File "/root/.local/share/hatch/env/virtual/datadog-mysql/bAxv122H/py3.11-5.7/lib/python3.11/site-packages/pymysql/cursors.py", line 153, in execute
    result = self._query(query)
             ^^^^^^^^^^^^^^^^^^
  File "/root/.local/share/hatch/env/virtual/datadog-mysql/bAxv122H/py3.11-5.7/lib/python3.11/site-packages/pymysql/cursors.py", line 322, in _query
    conn.query(q)
  File "/root/.local/share/hatch/env/virtual/datadog-mysql/bAxv122H/py3.11-5.7/lib/python3.11/site-packages/pymysql/connections.py", line 562, in query
    self._execute_command(COMMAND.COM_QUERY, sql)
  File "/root/.local/share/hatch/env/virtual/datadog-mysql/bAxv122H/py3.11-5.7/lib/python3.11/site-packages/pymysql/connections.py", line 843, in _execute_command
    raise err.InterfaceError(0, "")